app/assets/v2/js/pages/jobs.js

```diff
@@ -0,0 +1,672 @@
+        localStorage[key] += $(this).val() + ',';
+      });
+
+      // Removing the start and last comma to avoid empty element when splitting with comma
+      localStorage[key] = localStorage[key].replace(/^,|,\s*$/g, '');
+    }
+  }
+};
+
+// saves search information default
+var set_sidebar_defaults = function() {
+
+  // Special handling to support adding keywords from url query param
+  var q = getParam('q');
+  var keywords;
+
+  if (q) {
+    keywords = decodeURIComponent(q).replace(/^,|\s|,\s*$/g, '');
+
+    if (localStorage['keywords']) {
+      keywords.split(',').forEach(function(v, k) {
+        if (localStorage['keywords'].indexOf(v) === -1) {
+          localStorage['keywords'] += ',' + v;
+        }
+      });
+    } else {
+      localStorage['keywords'] = keywords;
+    }
+
+    window.history.replaceState(history.state, 'Issue Explorer | Gitcoin', '/explorer');
+  }
+
+  if (localStorage['order_by']) {
+    $('#sort_option').val(localStorage['order_by']);
+    $('#sort_option').selectmenu('refresh');
+  }
+
+  for (var i = 0; i < sidebar_keys.length; i++) {
+    var key = sidebar_keys[i];
+
+    if (localStorage[key]) {
+      if (key !== 'tech_stack') {
+        $('input[name=' + key + '][value=' + localStorage[key] + ']').prop('checked', true);
+      } else {
+        localStorage[key].split(',').forEach(function(v, k) {
+          $('input[name=' + key + '][value=' + v + ']').prop('checked', true);
+        });
+      }
+    }
+  }
+};
+
+var set_filter_header = function() {
+  var idxStatusEl = $('input[name=idx_status]:checked');
+  var filter_status = idxStatusEl.attr('val-ui') ? idxStatusEl.attr('val-ui') : 'All';
+
+  // TODO: See what all filters are to be displayed from designs
+
+  $('#filter').html('All');
+};
+
+var toggleAny = function(event) {
+  if (!event)
+    return;
+  // var key = event.target.name;
+  // var anyOption = $('input[name=' + key + '][value=any]');
+
+  // // Selects option 'any' when no filter is applied
+  // if ($('input[name=' + key + ']:checked').length === 0) {
+  //   anyOption.prop('checked', true);
+  //   return;
+  // }
+  // if (event.target.value === 'any') {
+  //   // Deselect other filters when 'any' is selected
+  //   $('input[name=' + key + '][value!=any]').prop('checked', false);
+  // } else {
+  //   // Deselect option 'any' when another filter is selected
+  //   anyOption.prop('checked', false);
+  // }
+};
+
+var getFilters = function() {
+  var _filters = [];
+
+  for (var i = 0; i < sidebar_keys.length; i++) {
+    var key = sidebar_keys[i];
+
+    $.each($('input[name=' + key + ']:checked'), function() {
+      if ($(this).attr('val-ui')) {
+        _filters.push('<a class="filter-tag ' + key + '"><span>' + $(this).attr('val-ui') + '</span>' +
+          '<i class="fa fa-times" onclick="removeFilter(\'' + key + '\', \'' + $(this).attr('value') + '\')"></i></a>');
+      }
+    });
+  }
+
+  if (localStorage['keywords']) {
```

Comment:
this will be a name collision with the `dashboard.js` which also has `localStorage['keywords`